### ISSUE TITLE
Mac app bundle does not open: LSOpenURLsWithRole() failed with error -10810

### ISSUE THREAD

**Issue by ogerardin** [fvarrui/JavaPackager#39]
The generated xxx.app bundle does not open. 

When double-clicked in Finder, icon bounces a few times in the dock then disappears with no error message displayed.
When launched from a terminal with "open xxx.app", I get the following error:
`LSOpenURLsWithRole() failed with error -10810 for the file xxx.app`

Here's the plugin config:
```
                   <plugin>
                        <groupId>io.github.fvarrui</groupId>
                        <artifactId>javapackager</artifactId>
                        <executions>
                            <execution>
                                <phase>package</phase>
                                <goals>
                                    <goal>package</goal>
                                </goals>
                                <configuration>
                                    <description>${app.description}</description>
                                    <displayName>${app.name}</displayName>
                                    <name>${app.name}</name>
                                    <organizationName>${app.copyright}</organizationName>
                                    <mainClass>${app.runnableMainClass}</mainClass>
                                    <runnableJar>${project.build.directory}/${app.runnableJar}</runnableJar>
                                    <copyDependencies>false</copyDependencies>
                                    <bundleJre>true</bundleJre>
                                    <jrePath>${java.home}</jrePath>
                                    <generateInstaller>true</generateInstaller>
                                    <macConfig>
                                        <!-- FIXME dmg temporarily disabled because it fails on Travis-CI -->
                                        <generateDmg>false</generateDmg>
                                        <generatePkg>true</generatePkg>
                                    </macConfig>
                                </configuration>
                            </execution>
                        </executions>
                    </plugin>
```

Running macOS Catalina 10.15.5.

**Comment by ogerardin** [comment]
After some investigation, problem occurs when there is a space in the app name.
Removing spaces from &lt;name&gt; makes it work.

The cause is $SCRIPTPATH not quoted properly in startup script:
```
Oliviers-MacBook-Pro:X-Plane Manager olivier$ cd X-Plane\ Manager.app/Contents/MacOS/
Oliviers-MacBook-Pro:MacOS olivier$ chmod a+x *
Oliviers-MacBook-Pro:MacOS olivier$ ./startup 
./startup: line 5: /Users/olivier/Projects/Idea/XPman/xpman-fx-dist/target/X-Plane: No such file or directory
```

Obsviouly the fix if to change 
`$SCRIPTPATH/universalJavaApplicationStub $@`
to
`"$SCRIPTPATH/universalJavaApplicationStub" $@`

I don't think I need to submit a PR for this :)

**Comment by fvarrui** [comment]
Hi @ogerardin!
You are right! Whitespaces are my headache! 

`universalJavaApplicationStub` call from `startup` script fixed on **v1.0.3-SNAPSHOT**.

Thanks!

**Comment by ogerardin** [comment]
It's everyone's headache :)
